### DISCUSSION TITLE
Welcome to global_tectonics Discussions!

### DISCUSSION THREAD

**Discussion by dhasterok** [dhasterok/global_tectonics#1] [Announcements]
## 👋 Welcome to global_tectonics!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas and offer model improvements.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dhasterok** [comment]
Hi World,

I'm Derrick Hasterok a Lecturer in Geophysics at the University of Adelaide and the maintainer of this repository.  The models you will find in this repository are the culmination of a three-years long project to develop a global geological province and tectonic plate map.  The goal was to make a set of seamless polygons that can be used as a data standard for geospatial analysis associated with geoscientific databases.  There are undoubtably inaccuracies and changes that should happen as a better understanding of crustal architecture.  We would like to your help to continually improve this model by leaving comments for discussion.  If you would like to be more involved, let us know that too.

**Comment by renatoparreira** [comment]
Hi, I downloaded the kml files but when opening, all of them, in Google Earth it gave an error, bellow:
«
Falha ao abrir o ficheiro "C:\Users\xxxx\Downloads\boundaries.kml": Erro durante a análise do ficheiro "C:/Users/xxxx/Downloads/boundaries.kml" na linha 9, coluna 0:
Elemento inesperado "html"
»
Thanks
